pkg/resources/server.go

```diff
@@ -180,6 +201,16 @@ func (rs *resourceServer) ListAndWatch(empty *pluginapi.Empty, stream pluginapi.
 	}
 }
 
+func (rs *resourceServer) updateCDISpec() error {
+	rs.resourcePool.GetDevices()
```

Comment:
Do you know if this call has any effect? It sounds like a "get" where the return value is discarded. If it has any side  effect, please add a comment

Reply:
`GetDevices` call is deleted now